common/types/custom_panels.ts

```diff
@@ -39,7 +52,7 @@ export interface SavedVisualizationType {
   user_configs: any;
 }
 
-export interface pplResponse {
+export interface PplResponse {
```

Comment:
Should we name this `PPLResponse`?

Reply:
I could.  This was a linter catch, inteface names must be PascalCase or UPPER_SNAKE_CASE.